tests/transformers/test_transformers_model_export.py

```diff
@@ -333,7 +333,7 @@ def zero_shot_pipeline():
 @flaky()
 def table_question_answering_pipeline():
     return transformers.pipeline(
-        task="table-question-answering", model="microsoft/tapex-base-finetuned-wtq"
+        task="table-question-answering", model="google/tapas-tiny-finetuned-wtq"
```

Comment:
Switching to a VERY lightweight up-to-date model from google here. The tiny size version isn't particularly capable (which is why the test validations below are modified and simplified), but we're not testing the capabilities of a model here.